frame-metadata/src/lib.rs

```diff
@@ -48,53 +40,45 @@ cfg_if::cfg_if! {
 use codec::{Encode, Output};
 
 /// A type that decodes to a different type than it encodes.
-#[cfg(any(
-	feature = "v13",
-	feature = "v12",
-	feature = "v11",
-	feature = "v10",
-	feature = "v9",
-	feature = "v8",
-	feature = "legacy"
-))]
+#[cfg(feature = "legacy")]
 pub mod decode_different;
 
 /// Metadata v8
-#[cfg(feature = "v8")]
+#[cfg(feature = "legacy")]
 pub mod v8;
 
 /// Metadata v9
-#[cfg(feature = "v9")]
+#[cfg(feature = "legacy")]
 pub mod v9;
 
 /// Metadata v10
-#[cfg(feature = "v10")]
+#[cfg(feature = "legacy")]
 pub mod v10;
 
 /// Metadata v11
-#[cfg(feature = "v11")]
+#[cfg(feature = "legacy")]
 pub mod v11;
 
 /// Metadata v12
-#[cfg(feature = "v12")]
+#[cfg(feature = "legacy")]
 pub mod v12;
 
 /// Metadata v13
-#[cfg(feature = "v13")]
+#[cfg(feature = "legacy")]
 pub mod v13;
 
 /// Metadata v14
-#[cfg(feature = "v14")]
+#[cfg(feature = "current")]
 pub mod v14;
 
 /// Metadata v15
-#[cfg(feature = "v15-unstable")]
+#[cfg(all(feature = "unstable", feature = "current"))]
```

Comment:
Should it be the other way round? ie if you enable the "unstable" feature you get v14 and v15 (since some types are shared). And if you enable "current" you just get v14 atm?

Reply:
Oh yea! It makes more sense to have the `unstable` feature enable the `current` than to need to specify them both, thanks! Have modified it